docker/packager/binary/build.sh

```diff
@@ -17,7 +17,9 @@ ccache --show-stats ||:
 ccache --zero-stats ||:
 ln -s /usr/lib/x86_64-linux-gnu/libOpenCL.so.1.0.0 /usr/lib/libOpenCL.so ||:
 rm -f CMakeCache.txt
-cmake --debug-trycompile --verbose=1 -DCMAKE_VERBOSE_MAKEFILE=1 -LA "-DCMAKE_BUILD_TYPE=$BUILD_TYPE" "-DSANITIZE=$SANITIZER" -DENABLE_CHECK_HEAVY_BUILDS=1 "$CMAKE_FLAGS" ..
+
+read -ra CMAKE_FLAGS <<< "${CMAKE_FLAGS:-}"
```

Comment:
Maybe write a comment? (Does it mean "read into array"?)
Or this trash script does not deserve a comment?

Reply:
I've never seen this syntax before but seems like bash spellchecker requires to use array syntax for flags...